projects/angular/src/accordion/accordion-panel.spec.ts

```diff
@@ -176,6 +181,13 @@ describe('ClrAccordionPanel', () => {
         accordionPanel.togglePanel();
         expect(component.change).toHaveBeenCalledWith(false);
       });
+
+      it('when [clrAccordionPanelHeadingEnabled] is not set at all, heading role should not be present', () => {
```

Comment:
Why is this in the "TypeScript API" test block instead of the "View" test block with other test? It's a test for an input and a DOM attribute (or lack thereof), so I think it should be moved.